05_syntax_lab/03.py

```diff
@@ -1,7 +1,10 @@
-"""
-Write a program that randomizes a number
-and prints the sum total of its digits.
-For example if the number was: 2345
-The result should be: 14
-"""
-
+#commited by Shlomit
+
+from random import randint
+num = randint(1, 10000)
+tmpnum = num

```

Comment:
Try to choose more descriptive variable names (as a general programming comment, not necessarily for python). For example `original` or `orig_value` or simply `input_number` would work better